server/src/domain/storage-template/storage-template.service.ts

```diff
@@ -76,6 +76,11 @@ export class StorageTemplateService {
 
   // TODO: use asset core (once in domain)
   async moveAsset(asset: AssetEntity, metadata: MoveAssetMetadata) {
+    if (asset.isReadOnly) {
+      this.logger.debug(`Not moving read-only asset: ${asset.originalPath}`);
```

Comment:
Maybe this should be a verbose log actually

Reply:
Changed this to verbose in my next code push.